content/chapters/io/lecture/demo/devices/hwaddr-ioctl.c

```diff
@@ -16,31 +16,36 @@
 
 #include "utils/utils.h"
 
-#define IFNAME			"enp0s31f6"
-
-int main(void)
+int main(int argc, const char *argv[])
 {
+	if (argc < 2) {
```

Comment:
Move this check after declaring your variables.

Reply:
Done.